.github/workflows/compliance.yml

```diff
@@ -47,5 +47,22 @@ jobs:
       - uses: actions/checkout@v4
       - name: Check license header
         uses: apache/skywalking-eyes/header@v0.6.0
-      - name: Check code style
+      - name: Check rust code style
         run: cd python && make check-rust
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python-version }}
```

Comment:
this won't work - but i can't push change to this file due to permission issue (disallow changing workflow in the fork). will need to fix in a separate PR. in this PR, this won't fail as the GH actions trigger is set to PR target.